Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -371,6 +378,11 @@ void RenderBox::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle
     if (isDocElementRenderer || isBodyRenderer) {
 #if ENABLE(DARK_MODE_CSS)
         view().frameView().recalculateBaseBackgroundColor();
+
+        if (oldStyle && oldStyle->colorScheme() != newStyle.colorScheme()) {
+            view().frameView().invalidateScrollCorner(view().frameView().scrollCornerRect());
+            view().frameView().invalidateScrollbarsForAllScrollableAreas();
+        }
```

Comment:
I think this won't work if you change the `color-scheme` of a specific-element? Since it's underneath `isDocElementRenderer || isBodyRenderer`.

Reply:
Oh you might be right actually. I'll add the other line back to the above code snippet. Won't be at a computer till Thursday so apologies for slow response.

Reply:
No worries! Appreciate your work on this 😄